packages/mattermost-redux/src/selectors/entities/posts.test.ts

```diff
@@ -983,9 +1003,9 @@ describe('Selectors.Posts', () => {
                         },
                     },
                 },
-            };
+            } as unknown as GlobalState;
 
-            assert.deepEqual(getPostIdsAroundPost(state, 'a', '1234'), ['a']);
+            expect((getPostIdsAroundPost as any)(state, 'a', '1234')).toEqual(['a']);
```

Comment:
the selector maker should have had its fourth param as optional, as well as the keys in its object, because they are defaulted. Changed now, and removed the as any coercions.